lib/util.rb

```diff
@@ -70,8 +70,8 @@ def self.create_certificate(subject:, duration:, extensions: [], issuer_cert: ni
     [cert, key]
   end
 
-  def self.exception_to_hash(ex)
-    {exception: {message: ex.message, class: ex.class.to_s, backtrace: ex.backtrace, cause: ex.cause.inspect}}
+  def self.exception_to_hash(ex, backtrace: true)
+    {exception: {message: ex.message, class: ex.class.to_s, backtrace: backtrace ? ex.backtrace : nil, cause: ex.cause.inspect}}
```

Comment:
I think this style is simpler:

```suggestion
  def self.exception_to_hash(ex, backtrace: ex.backtrace)
    {exception: {message: ex.message, class: ex.class.to_s, backtrace:, cause: ex.cause.inspect}}
```